app/SectionApp/TodoContainer/Tasks/CreateStepTask.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\SectionApp\TodoContainer\Tasks;
+
+use App\SectionApp\TodoContainer\Repositories\Interfaces\StepRepositoryInterface;
+use App\SectionApp\TodoContainer\Values\CreateOrUpdateStepValue;
+use Spatie\LaravelData\DataCollection;
+
+/**
+ * Class CreateStepTask
+ *
+ * This class is responsible for creating a new step
+ */
+class CreateStepTask
```

Comment:
Mình tạo nhiều steps cho nên thêm `s` zô nha, update lại comment class nữa
```suggestion
class CreateStepsTask
```